data/README.md

```diff
@@ -62,3 +62,22 @@ wp.data.query( select => {
 	};
 } )( Component );
 ```
+
+
+### `wp.data.subscribe( listener: func )`
+
+Function used to subscribe to changes to data changes. The listener function is called each time a change is made to any of the registered reducers. This function returns a `unsubscribe` function used to abort the subscription.
```

Comment:
It might read better if we change `subscribe to changes to data changes` -> `subscribe to data changes`.

Reply:
🙃